evap/development/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from evap.development import views
+
+urlpatterns = [
+    path("components", views.development_components, name="development_components"),
```

Comment:
I would suggest `dev_helpers` as a more obvious name, but maybe that's just personal preference.

Reply:
@janno42 your opinion here? :cat:

Reply:
helpers sounds good to me, but components is also fine :)